### DISCUSSION TITLE
folders with images (duplicates) not showing the images?

### DISCUSSION THREAD

**Discussion by pooh22** [photoprism/photoprism#2663] [Q&A]
Hi,

I have a couple of directories with copies of images (e.g. a collection of holiday photos). Obviously these are duplicates and thus probably stacked during the indexing. I'm not sure if "not showing duplicate photos" is a feature or a bug. In my particular case it surely is a bug, because I can't browse this "folder" with my collection, as it contains only 1 image which is probably one that has been modified by me, not the hundreds of other images which are also elsewhere in the storage.

For me it would be fixed if the stacked images show up everywhere they are actually stored in the folders view. I suppose that would mean a stacked image should have a list of "showing up" locations associated with it...

Cheers

/Simon

**Comment by graciousgrey** [comment]
Duplicate detection is a feature not a bug.

Duplicates are skipped, related files get stacked see https://docs.photoprism.app/user-guide/library/duplicates/

Open issues that might be of interest for you: 
- https://github.com/photoprism/photoprism/issues/1308
- https://github.com/photoprism/photoprism/issues/1306

**Reply by pooh22** [reply]
Thanks for the suggestions, I suppose there are lots of different ways to look at handling duplicates. In my case, I am using/trying photoprism as a view on my existing photo repository, which is file based and contains duplicates due to quick and dirty organizing actions over the past 22 years. I usually manage my photo's using digikam and geeqie, which don't have the same duplicates handling (if at all).

I don't mean to label "duplicate detection as a bug", of course it's a nice feature! The "bug" in this case would be not being able to see the images (duplicates) I placed in physical folder for viewing a usable subset of holiday images.

I would also be helped by having a way to create "albums" from selecting a combinations of tags and including all images where the tags are set (or even a boolean role like: "tag:holiday and tag:germany and tag:ablum and not tag:documentation"

Something like that would dynamically define an album from the images themselves and duplicates would not be necessary or a problem.

Cheers

/Simon

**Reply by graciousgrey** [reply]
Smart albums are planned https://github.com/photoprism/photoprism/issues/1431 :)
